### PR TITLE
SAK-31456 - Collapsible side-tools menu won't stayed minimized after page refreshes

### PR DESCRIPTION
**Pull request by fgeraci**
[SAK-31456](https://jira.sakaiproject.org/browse/SAK-31456)

### CODE REVIEW

File: reference/library/src/morpheus-master/js/src/sakai.morpheus.toggletools.js [hunk 1]
```diff
@@ -10,22 +10,43 @@ function toggleMinimizeNav(){
...
 
   el.toggleClass('min max');
-
-  if (label.text() == el.data("title-expand")) {
-    label.text(el.data("text-original"));
+  
+  if (label.text() == el.data("title-expand") || collapsed) {
+	document.cookie = "sakai_nav_minimized=false; path=/";
+	collapsed = false;
+	console.log('cookie set to false');
```
Comment (juanjmerono) [inline reference/library/src/morpheus-master/js/src/sakai.morpheus.toggletools.js]: Remove console log here.

File: reference/library/src/morpheus-master/js/src/sakai.morpheus.toggletools.js [hunk 2]
```diff
@@ -10,22 +10,43 @@ function toggleMinimizeNav(){
...
     el.attr('title', (el.data("text-original")));
     el.attr('aria-pressed', true);
-    document.cookie = "sakai_nav_minimized=false; path=/";
-
   } else {
-
-    el.data("text-original", label.text());
+	document.cookie = "sakai_nav_minimized=true; path=/";
+	collapsed = true;
+	console.log('cookie set to true');
```
Comment (juanjmerono) [inline reference/library/src/morpheus-master/js/src/sakai.morpheus.toggletools.js]: Remove console log here.

**Comment by juanjmerono** [comment]
We needed !! Thanks @fgeraci !!

**Comment by fgeraci** [comment]
@juanjmerono pfffff how did I miss those?! XD - Thank you.

**Comment by SedueRey** [comment]
LGTM

**Pull request merged**